package.json

```diff
@@ -31,18 +31,21 @@
     "react-router-config": "^5.1.1",
     "react-router-dom": "^5.1.2",
     "react-router-query-params": "^1.0.4",
-    "react-scripts": "3.4.4",
+    "react-scripts": "^4.0.0",
     "react-world-flags": "^1.4.0",
     "typescript": "^3.7.0"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "jest",
+    "test": "react-scripts test",
```

Comment:
Since we have not ejected from CRA yet, it's simpler to let it do its thing
https://create-react-app.dev/docs/running-tests/#command-line-interface

This way, `npm test` runs the tests in "watch" mode on dev environment

We can probably set `CI=true` for the CI?
https://create-react-app.dev/docs/running-tests/#continuous-integration

Reply:
Yep, it's already there in .github/workflows/build_web.yml:

    - name: Test 
      run: npm test
      env:
        CI: true

Reply:
awesome! 
I actually didn't know yet how we setup the CI, great to know that it's in this file :)